makeself/makeself/test/run-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eu
+THIS="$(readlink -f "$0")"
+THISDIR="$(dirname "${THIS}")"
+
+# Run every available test
+
+echo ">> env:"
+uname -a
+
+cd "$THISDIR"
+
+bash "./run-testlinter.sh"
+
+for test in ./*test;
+do
+	echo "::group::$test"
+	bash "./$test" || { echo "*** ERROR: Test '$test' failed!"; exit 1; }
```

Comment:
We may want to modify this so that it uses `$BENCHMARK_SHELL` instead of `bash`?